utilities/validations.py

```diff
@@ -22,6 +22,8 @@ def validate_empty_fields(**kwargs):
     """
     for field in kwargs:
         value = kwargs.get(field)
+        if isinstance(value, str):
+            value = value.strip()  # remove white space
```

Comment:
awesome work @verenceLola though I think we should do away with the inline comment.

Reply:
Thanks @dnuwa. Let me work on that.